src/extensions/default/JavaScriptCodeHints/styles/brackets-js-hints.css

```diff
@@ -21,6 +21,200 @@
+    color: cornflowerblue;
+    content: '12';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    visibility: hidden;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-number:before {
+    color: cornflowerblue;
+    content: '12';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-string:before {
+    color: cornflowerblue;
+    content: 'ab';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-function:before {
+    color: cornflowerblue;
+    content: 'fn( )';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-array:before {
+    color: cornflowerblue;
+    content: '[  ]';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-object:before {
+    color: cornflowerblue;
+    content: '{ :}';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-num-array:before {
+    color: cornflowerblue;
+    content: '[12]';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-string-array:before {
+    color: cornflowerblue;
+    content: '[ab]';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints.type-boolean:before {
+    color: cornflowerblue;
+    content: 'bool';
+    border: 1px cornflowerblue solid;
+    float:left;
+    width:25px;
+    text-align: center;
+    margin-left:-5px;
+    margin-right:5px;
+    line-height: 1.2em;
+}
+
+.brackets-js-hints-type-details {
+    color: #d3d3d3 !important;
+    font-weight: 100;
+    font-style: italic !important;
+    margin-right: 5px;
+    float:right;
+}
+
+.jshint-description {
+    display: none;
+    padding-left: 35px !important;
+    padding-right:10px !important;
+    color: #d4d4d4;
+    word-wrap: break-word;	
+    white-space: normal;
+    width:400px;
+    box-sizing: border-box;
+}
+
+.dark .jshint-description {
+    color: #696969;
+}
+
+.jshint-jsdoc {
+    display: none;
+    padding-left: 28px !important;
+    padding-right: 10px !important;
+    color: grey;
```

Comment:
It doesn't look very visible. We use `#ccc` in pref hints so it will be good to use it here as well.